environments/node.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
change file name back to FE / BE. Feel free to create a node that inherits this if you'd like.